src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    protected static String[] readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        List<String> statistic = null;
+        try {
+            statistic = Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot read file " + fromFileName, e);
+        }
+        String[] array = new String[statistic.size()];
+        array = statistic.toArray(array);
+        return array;
+    }
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    protected static String createReport(String fromFileName) {
```

Comment:
you'll have to modify this method according to previous comment

Reply:
done

Reply:
then use private